### PR TITLE
New logic for plot limits

### PR DESCRIPTION
**Pull request by cosimolupo**
Changed the logic on plot limits (in config files, 'None' is now accepted as value for t_start/t_stop to plot the entire signal).

### CODE REVIEW

**Comment by mdenker** [comment]
@cosimolupo Possibly already implemented. Please check and close if duplicate.

**Comment by mdenker** [comment]
Possibly new PR for the change in pipeline/utils/parse.py ?

File: cobrawap/pipeline/configs/config_template.yaml [hunk 1]
```diff
@@ -19,8 +19,8 @@ NEO_FORMAT: 'nix'
 USE_LINK_AS_STAGE_OUTPUT: True
 
 # Plotting parameters
-PLOT_TSTART: 0  # in s
-PLOT_TSTOP: 10  # in s
+PLOT_TSTART: 0  # float (in s) or 'None' -> T_START of the input signal is used
+PLOT_TSTOP: 10  # float (in s) or 'None' -> T_STOP of the input signal is used
```
Comment (mdenker) [inline cobrawap/pipeline/configs/config_template.yaml:23]: Here and in the following, does `T_START` refer to `t_start` of the Neo object? In that case I would suggest
```suggestion
PLOT_TSTART: 0  # float (in s) or 'None' -> t_start of the input signal is used
PLOT_TSTOP: 10  # float (in s) or 'None' -> t_stop of the input signal is used
```

**Review by mdenker — APPROVED** [review]
From what I can judge, this seems fine to me.

File: cobrawap/pipeline/configs/config_template.yaml (hunk 1 above)
Comment (cosimolupo) [reply cobrawap/pipeline/configs/config_template.yaml:23]: Ok, I will fix that everywhere!

**Comment by cosimolupo** [comment]
@mdenker I adapted the description in config template files, so to make more explicit and clear what means setting PLOT_TSTART and/or PLOT_TSTOP value to 'None'.

**Comment by cosimolupo** [comment]
Merging Pull Request #61 added some extra spacings, which I fixed in a further commit.

**Comment by mdenker** [comment]
Merge as soon as version number is increased, e.g., after #62 #63

**Review by rgutzen — APPROVED** [review]

**Review by gulpgiulia — APPROVED** [review]

**Pull request merged**